src/styles/common/icons.styl

```diff
@@ -0,0 +1,41 @@
+@font-face
+  font-family: "untitled-font-6"
```

Comment:
I think I got couple of comments and questions here:

1. Could you please rename `untitled-font-6` into something that would make more sense? Like "icons" or "glyphs".
2. Why do we need so many different font formats (i.e. `eot`, `woff`, `ttf`, `svg`)? I'd prefer to pick one and go with it.
3. What's the license of this font? Are we allowed to distribute it? Are we allowed to use it without a license notice?

Reply:
About the license, since these icons were generated by http://fontastic.me/, seems it's free to use without any additional license